core/dbt/cli/flags.py

```diff
@@ -394,6 +396,14 @@ def fire_deprecations(self):
 
         fire_buffered_deprecations()
 
+        # Handle firing deprecations of CLI aliases separately using argv
+        # because click because it difficult to disambiguite which CLI option was used
```

Comment:
Nit
```suggestion
        # because click makes it difficult to disambiguite which CLI option was used
```
The wording "because click because" just felt a little odd